servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/AbstractStreamingHttpConnection.java

```diff
@@ -90,7 +90,9 @@ abstract class AbstractStreamingHttpConnection<CC extends NettyConnectionContext
         maxConcurrencySetting = from(new IgnoreConsumedEvent<>(maxPipelinedRequests))
                 .concat(connection.onClosing())
                 .concat(succeeded(ZERO_MAX_CONCURRENCY_EVENT))
-                .multicast(1); // Allows multiple Subscribers to consume the event stream.
+                .replay(1); // Allow multiple Subscribers to consume, new Subscribers get last signal.
```

Comment:
Do we need `replay` here? Looks like it already supports multiple subscribers and can "replay" the same events.

Reply:
Replay makes it such that any new subscriber always gets the "latest" onNext signal (otherwise they may see nothing).

Reply:
But not in this case. This `maxConcurrencySetting` publisher is always replayable even without `replay` operator

Reply:
oh I see. `from()` `connection.onClosing()` and `succeeded(..)` all support multiple subscribers and will deliver events from the beginning. I will add a comment bcz if this changes it may subtly break w/out replay.